Misc/NEWS.d/next/Library/2020-09-13-02-02-18.bpo-39337.L3NXTt.rst

```diff
@@ -0,0 +1,2 @@
+similar to :c:func:`_Py_normalize_encoding`,
+:func:`encodings.normalize_encoding` should ignore non-ASCII letters.
```

Comment:
Please don't mention private functions which end users are not aware of.

Just say something like:

```
 :func:`encodings.normalize_encoding` now ignores non-ASCII letters. 
```

Same remark for the unit test docstring.

Reply:
make sense, I will update it.